aspnetcore/data/ef-rp/concurrency.md

```diff
@@ -322,19 +322,19 @@ The Delete page detects concurrency conflicts when the entity has changed after
 * A DbUpdateConcurrencyException exception is thrown.
 * `OnGetAsync` is called with the `concurrencyError`.
 
-### Update the Delete Razor page
+### Update the Delete page
 
 Update *Pages/Departments/Delete.cshtml* with the following code:
 
-[!code-html[](intro/samples/cu30/Pages/Departments/Delete.cshtml?highlight=1,10,39,51)]
+[!code-html[](intro/samples/cu30/Pages/Departments/Delete.cshtml?highlight=1,10,39,42,51)]
```

Comment:
While you're here, let's get the dev lang fixed too.

```suggestion
[!code-cshtml[](intro/samples/cu30/Pages/Departments/Delete.cshtml?highlight=1,10,39,42,51)]
```

Reply:
@scottaddie Awesome, thanks. I checked the rest of that topic and found a few more of those.